tests/engine/test_work_chain.py

```diff
@@ -1315,7 +1315,7 @@ def test_nested_expose(self):
     def test_issue_1741_expose_inputs(self):
```

Comment:
I think this regression test should still use the "bare" ``Int(5).store()`` - issue #1741 seems like it wouldn't occur with the ``lambda`` version regardless of whether the issue is fixed.

See https://docs.pytest.org/en/latest/warnings.html for how to filter the warning from the test run.

Reply:
Hmm very good point, however, if I use a stored node here, won't we again have the issues in unit testing with having node references in memory whose database counterpart has been deleted? So does this leave use between a rock and a hard place?

Reply:
Well, yes, but in this instance the whole cycle (class definition + usage) is defined _within_ a single test. So, it shouldn't be deleted in between. Maybe we just need to add a comment to explain this?

Reply:
Added a comment explaining it and marked the test to ignore the warning

Reply:
Fantastic!